.markdownlint-cli2.yaml

```diff
@@ -2,3 +2,8 @@ config:
   line-length: false
   no-emphasis-as-header: false
   first-line-heading: false
+paths:
+ - "**/md"
```

Comment:
This is different from what was in the command line. Did you verify it still works?

Reply:
well the lint is clear, but this should be `**/*.md`. Will fix it now.

Reply:
With fix applied:
```
markdownlint-cli2 v0.6.0 (markdownlint v0.27.0)
Finding: **/*.md !pkg !**/node_modules/*
Linting: 21 file(s)
Summary: 0 error(s)
```